README.md

```diff
@@ -89,28 +86,27 @@ By default VERSION is blank if you do not use the creator-platform-default.confi
 - openwrt-$(VERSION)-pistachio-pistachio_$(PROFILE)-uImage
 - openwrt-$(VERSION)-pistachio-pistachio_$(PROFILE)-uImage-initramfs
 - openwrt-$(VERSION)-pistachio-marduk-$(PROFILE)-rootfs.tar.gz
```

Comment:
If it is going to be all about cascoda only, then why is there even a need of this?  You can simply start below the ca8210 specific filenames and references.

Also, please get rid of the "Where PROFILE can be marduk_cc2520, marduk_ca8210, marduk_cc2520_wifi and VERSION is whatever mentioned in CONFIG_VERSION_NUMBER."  CC2520 doesn't need any specific mentioning, and the cc2520_wifi profile has been deleted.  Default ca8210 and cc2520 now have the Wi-Fi test related packages enabled